main/docs/secure/multi-factor-authentication/adaptive-mfa/customize-adaptive-mfa.mdx

```diff
@@ -615,7 +615,14 @@ Auth0 provides two Action templates based on Adaptive MFA for you to customize:
 
 ### Adaptive MFA template
 
-This template provides an example and starting point for how to build a custom business flow using individual risk assessments.
+This template provides an example and starting point for how to build a custom business flow using individual risk assessments. In this example, we use:
+
+* The [`api.multifactor.enable`](/docs/customize/actions/explore-triggers/signup-and-login-triggers/login-trigger/post-login-api-object#api-multifactor) Action trigger to handle both enrollment and issues configured MFA challenges at the end of the login flow.
+* The [`event.user.multifactor`](/docs/customize/actions/explore-triggers/signup-and-login-triggers/login-trigger/post-login-event-object) Actions trigger with the user's enrolled factors.
```

Comment:
"Actions trigger" or "Action trigger"? (not sure which one is the right way to refer to the product)